src/Files.App/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -96,6 +96,13 @@ public async Task<(ReturnResult, IStorageItem)> CreateAsync(IStorageItemWithPath
 			var progress = new Progress<FileSystemProgress>();
 			progress.ProgressChanged += (s, e) => returnStatus = returnStatus < ReturnResult.Failed ? e.Status!.Value.ToStatus() : returnStatus;
 
+			if (string.IsNullOrWhiteSpace(source.Name)
+				|| ContainsRestrictedCharacters(source.Name)
+				|| ContainsRestrictedFileName(source.Name))
+			{
+				return (ReturnResult.Failed, null);
```

Comment:
Does this display a dialog?

Reply:
No, but in CreateAsync no error displays a dialog

Reply:
Should I change it? @yaira2

Reply:
What does the error dialog look like?

Reply:
The error dialog could be something like this, if we use existing strings already in resources.resw
![image](https://user-images.githubusercontent.com/55754091/211636448-5c118320-119c-4e63-8786-a3b3fef9b241.png)

Reply:
Strings would have to be modified but this is the right idea.

Reply:
Ok, I'll fix it now

Reply:
Done